langfuse/api/resources/span/types/update_span_request.py

```diff
@@ -12,6 +12,7 @@
 class UpdateSpanRequest(pydantic.BaseModel):
     span_id: str = pydantic.Field(alias="spanId")
     trace_id: typing.Optional[str] = pydantic.Field(alias="traceId")
+    name: typing.Optional[str]
```

Comment:
Good catch! All files in langfuse/api are auto-generated by fern. Added a PR on the core project and will update this PR as well https://github.com/langfuse/langfuse/pull/101

By keeping the openapi docs in core in sync, we can also update the TS SDK accordingly.

Reply:
Alright